inscriptions/ISic020537.xml

```diff
@@ -0,0 +1,225 @@
+                        </objectDesc>
+                        <handDesc>
+                            <handNote>
+                                <!-- Based on Glyph table: -->
+                                <locus from="line1" to="line1">Line 1</locus>
+                                <dimensions type="letterHeight">
+                                    <height unit="mm"/>
+                                </dimensions>
+                            </handNote>
+                        </handDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origPlace>
+                                <placeName type="ancient" ref="https://pleiades.stoa.org/places/462214">Gela</placeName>
+                                <placeName type="modern">Gela</placeName>
+                                <geo>37.0630925, 14.2584705</geo>
+                            </origPlace>
+                            <origDate datingMethod="#julian" notBefore-custom="-0450" notAfter-custom="-0450">450 BC</origDate>
+                        </origin>
+                        <provenance type="found">Found in Gela<geo>37.0630925, 14.2584705</geo>.</provenance>
+                        <provenance type="observed" subtype="autopsied">No Autopsy</provenance>
+                        <acquisition>Gela, Museo Archeologico Regionale di Gela</acquisition>
+                    </history>
+                </msDesc>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>Encoded following EpiDoc guidelines 9.2</p>
+            <xi:include href="../alists/ISicily-taxonomies.xml">
+                <xi:fallback>
+                    <p>Taxonomies for ISicily controlled values</p>
+                </xi:fallback>
+            </xi:include>
+        </encodingDesc>
+        <profileDesc>
+            <calendarDesc>
+                <calendar xml:id="julian">
+                    <p>Julian Calendar</p>
+                </calendar>
+            </calendarDesc>
+            <langUsage>
+                <language ident="en">English</language> 
+                <language ident="it">Italian</language> 
+                <language ident="grc">Ancient Greek</language> 
+                <language ident="la">Latin</language> 
+                <language ident="he">Hebrew</language> 
+                <language ident="phn">Phoenician</language>
+                <language ident="xpu">Punic</language>
+                <language ident="osc">Oscan</language> 
+                <language ident="xly">Elymian</language> 
+                <language ident="scx">Sikel</language>  
+                <language ident="sxc">Sikan</language>  
+            </langUsage>
+            <textClass>
+                <keywords scheme="http://www.eagle-network.eu/voc/typeins.html">
+                    <term ana="#function.illustrative" ref="???">illustrative (exhortation)</term> <!-- something to discuss -->
```

Comment:
indeed, tricky. My indication would be to treat it as another variant on 'greeting' or 'salutation' at this point, since it's ultimately not dissimilar. There's nothing in Eagle that covers it at present. Very happy to discuss ;-)